netty/src/main/java/io/grpc/netty/NettyServerBuilder.java

```diff
@@ -189,10 +190,21 @@ public NettyServerBuilder channelFactory(ChannelFactory<? extends ServerChannel>
    * Specifies a channel option. As the underlying channel as well as network implementation may
    * ignore this value applications should consider it a hint.
    *
+   * @since 1.28.1
```

Comment:
`1.29.0` branch cut is already done. it should `1.30.0`
```suggestion
   * @since 1.30.0
```